Bottom: d4edc2731701b6ca6a94f9087e44dbb9f76ec25c
Top:    a60724bf5aa9081e8c88db5b5cfd3786af42b333
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-18 11:31:12 -0700

Refresh of fork

---

diff --git a/src/preload.c b/src/preload.c
index 56c6c7d..731643d 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -471,9 +471,8 @@ static int connect_fork(int socket, const struct sockaddr *addr, socklen_t addrl
 	flags = real.fcntl(fd, F_GETFD);
 	real.fcntl(fd, F_SETFD, 0);
 	ret = real.connect(fd, addr, addrlen);
-	if (ret) {
+	if (ret)
 		return ret;
-	}
 
 	ret = real.recv(fd, &msg, sizeof msg, MSG_PEEK);
 	if ((ret != sizeof msg) || msg) {
